docs/getting-started.md

```diff
@@ -230,38 +230,69 @@ Now run `npm start` again and open http://localhost:8080. You should see a Mater
 
 <img src="button.png" alt="Button" width="90" height="36">
 
-### Step 3: Webpack with ES2015
+### Step 3: Installing JavaScript packages
 
-We need to configure webpack to bundle ES2015 JavaScript into standard JavaScript, through [babel](https://babeljs.io). You’ll need all of these dependencies:
+Before we configure Webpack you’ll need all of these dependencies:
 
 - [babel-core](https://www.npmjs.com/package/babel-core)
 - [babel-loader](https://www.npmjs.com/package/babel-loader): Compiles JavaScript files using babel
+- [ts-loader](https://www.npmjs.com/package/ts-loader): Compiles TypeScript files
+- [typescript](https://www.npmjs.com/package/typescript): TypeScript compiler
 - [babel-preset-es2015](https://www.npmjs.com/package/babel-preset-es2015): Preset for compiling es2015
 - [babel-plugin-transform-object-assign](http://npmjs.org/package/babel-plugin-transform-object-assign), for IE 11 support
 
+> NOTE: You can decide to leave out `ts-loader` and `typescript` if you do not want to use TypeScript. However we do recommend you using TypeScript as it provides all the features of type safety. You can also use other TypeScript loaders such as [Awesome TypeScript Loader](https://www.npmjs.com/package/awesome-typescript-loader). We chose ts-loader because it is what we use to build our components.
+
 You can install all of them by running this command:
 
 ```
-npm install --save-dev babel-core@6 babel-loader@7 babel-preset-es2015 babel-plugin-transform-object-assign
+npm install --save-dev babel-core@6 babel-loader@7 babel-preset-es2015 babel-plugin-transform-object-assign ts-loader@3.5.0
```

Comment:
Should typescript also be installed locally? It's not in this list...

Also, babel-plugin-transform-object-assign should no longer be needed (we should test that - I'm pretty sure all instances of Object.assign should no longer exist on the feat/typescript branch)

Reply:
Hmm...ya i suppose it should. Below I also recommend installing globally, but it should be included in their deps in package.json